File: src/main/java/app/askresume/api/extract/dto/response/ExtractedTextResponse.java
package app.askresume.api.extract.dto.response;

import io.swagger.v3.oas.annotations.media.Schema;

public record ExtractedTextResponse(
        @Schema(description = "변환된 TEXT", example = "Hello, I'm backend developer Hong Gildong. Something, something, something...")
        String text
) {
}

File: src/main/kotlin/app/askresume/api/extract/dto/response/ExtractedTextResponse.kt
package app.askresume.api.extract.dto.response

import io.swagger.v3.oas.annotations.media.Schema

data class ExtractedTextResponse(

    @field:Schema(
        description = "변환된 TEXT",
        example = "Hello, I'm backend developer Hong Gildong. Something, something, something..."
    )
    val text: String,

)